apps/member-profile/app/shared/utils/format.utils.ts

```diff
@@ -25,3 +27,33 @@ export function formatName({
     ? `${firstName} (${preferredName}) ${lastName}`
     : `${firstName} ${lastName}`;
 }
+
+/**
+ * Formats a rejection reason into a user-friendly message.
+ *
+ * @param {string} reason - The raw rejection reason.
+ * @returns {string} A formatted, user-friendly explanation of the rejection reason.
+ *
+ * @example
+ * // Returns "This referral was rejected because an ineligible major."
+ * formatRejectionReason('ineligible_major');
+ *
+ **/
+export function formatRejectionReason(reason: string) {
+  if (!reason) return 'Something happened.';
+
+  switch (reason) {
+    case ApplicationRejectionReason.BAD_LINKEDIN:
+      return 'Incorrect or suspicious LinkedIn';
```

Comment:
We might want to change the wording to just 'Incorrect LinkedIn' or maybe 'Incomplete LinkedIn'-using the word 'suspicious' could be problematic lol